Gems/ROS2/Code/Include/ROS2/Lidar/LidarSystemBus.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Component/EntityId.h>
+#include <AzCore/EBus/EBus.h>
+
+namespace ROS2
+{
+    class LidarSystemRequests
+    {
+    public:
+        AZ_RTTI(LidarSystemRequests, "{007871d1-2783-4382-977b-558f436c54a5}");
+        virtual ~LidarSystemRequests() = default;
+
+        //! Creates a new Lidar.
+        //! @param lidarEntityId EntityId holding the ROS2LidarSensorComponent.
+        //! @return A unique Id of the newly created Lidar.
+        virtual AZ::Uuid CreateLidar(const AZ::EntityId& lidarEntityId) = 0;
```

Comment:
nit: As EntityIds are just a wrapper around a `u64` it's better to just pass by value so this can `EntityId lidarEntityId`

Reply:
Thanks! Fixed.